js/plugins/vertexai/src/embedder.ts

```diff
@@ -370,13 +370,13 @@ export function defineVertexAIEmbedder(
             // Text only embedder
             instance = {
               content: doc.text,
-              task_type: options?.taskType,
-              title: options?.title,
+              task_type: embedderRef.config?.taskType,
```

Comment:
I'm not sure about reading from the embedderRef here?

Before we used `options` - i'm thinking at request time we could have these options specified which would take precedence over the config of the ref maybe